src/main/java/org/adridadou/ethereum/propeller/EthereumProxy.java

```diff
@@ -222,15 +229,18 @@ private String printTypes(Object[] constructorArgs) {
         Objects.requireNonNull(txHash);
         long currentBlock = eventHandler.getCurrentBlockNumber();
 
-        Observable<TransactionInfo> droppedTxs = eventHandler.observeTransactions()
+        Flowable<TransactionInfo> droppedTxs = eventHandler.observeTransactions()
+                .toFlowable(BackpressureStrategy.BUFFER)
                 .filter(params -> params.getReceipt().map(receipt -> Objects.equals(receipt.hash, txHash))
                         .orElse(false) && params.getStatus() == TransactionStatus.Dropped);
 
-        Observable<TransactionInfo> timeoutBlock = eventHandler.observeBlocks()
+        Flowable<TransactionInfo> timeoutBlock = eventHandler.observeBlocks()
+                .toFlowable(BackpressureStrategy.BUFFER)
                 .filter(blockParams -> blockParams.blockNumber > currentBlock + config.blockWaitLimit())
-                .map(params -> null);
+                .map(blockInfo -> new EmptyTransactionInfo());
```

Comment:
The emitters from rxjava2 doesn't allow to map null elements anymore. So I've changed to send an empty info and then we filter that out downstream.